server/src/main/java/org/spine3/server/event/enrich/EventEnrichmentsMap.java

```diff
@@ -66,6 +81,13 @@ private EventEnrichmentsMap() {}
 
     private static class Builder {
 
+        /**
+         * Constant indicating a package qualifier.
+         */
+        private static final String PACKAGE_WILDCARD_INDICATOR = ".*";
+
+        private static final String WILDCARD_TYPE_UPON_FIELD_INDICATOR = "*.";
```

Comment:
The name is not clear enough. Please add a documentation for it. Maybe you'll find a better name while writing the docs.